commands/build.js

```diff
@@ -155,13 +156,20 @@ module.exports = {
         data: JSON.stringify({
           name,
           description,
+          repository,
           xr_type: xrType,
           start_url: startUrl,
           xr_details: xrDetails,
         }, null, 2),
       },
     ];
 
+    // Default to package name or a.wbn if not explicitly provided
+    argv.output = argv.output || `${name}.wbn` || 'a.wbn';
```

Comment:
`a.wbn` will never be hit here. I think the intent was to check if `name` exists?

Reply:
Thanks! I've removed the final `|| a.wbn`. 

It looks like `name` should always be defined by this point, because if none of the earlier `if`s that set the name were satisfied, we will have thrown an error or just returned early.